src/peppr/metric.py

```diff
@@ -1109,11 +1150,19 @@ def evaluate(self, reference: struc.AtomArray, pose: struc.AtomArray) -> float:
         if reference.array_length() == 0 or pose.array_length() == 0:
             return np.nan
 
-        # Only evaluate on PLI systems - check for both ligands and proteins
-        ligand_mask = reference.hetero
-        protein_mask = ~ligand_mask
+        reference, pose = _match_receptors_only(reference, pose)
```

Comment:
for counting plifs we definitely need consistent residue ids that are matched between reference and pose but do we need to match the reference and pose and align them in 3D space, since we compare local interactions we can enumerate the types of interactions for the pose and reference in completely different orientations and still report the score ?